Formula/p/pkl.rb

```diff
@@ -0,0 +1,47 @@
+class Pkl < Formula
+  desc "CLI for the Pkl programming language"
+  homepage "https://pkl-lang.org"
+  url "https://github.com/apple/pkl/archive/refs/tags/0.25.2.tar.gz"
+  sha256 "810f6018562ec9b54a43ba3cea472a6d6242e15da15b73a94011e1f8abc34927"
+  license "Apache-2.0"
+
+  env :std
+  depends_on "openjdk@17" => :build
```

Comment:
Does it not work with modern OpenJDK?

Reply:
as I recall this is the standard for internal Apple devtools; I think AppleJDK might be based on 17

Reply:
But does it work with a newer version?

Reply:
No, since Homebrew software should run on the newest dependencies unless they are broken. So it should use `openjdk` unless it is known not to work with it.

Reply:
Changed to just `openjdk`